### ISSUE TITLE
Phone Mock Status Bar

### ISSUE THREAD

**Issue by NilsBaumgartner1994** [flameddd/screenshots-ci-action#37]
So when trying to automate our app screenshots for the store we are missing the status bar. We would like to integrate the status bar for the screenshots. Any way to add this?

**Comment by flameddd** [comment]
What's `status bar`, I not sure what it is.

Can you provide some example image to me ? 

If it's OUT of viewpoint, I'm afraid, not too much thing `puppeteer` able to do ...  

[Screen_Capture_API](https://developer.mozilla.org/en-US/docs/Web/API/Screen_Capture_API) ? maybe, but, usually it has very strict permission control.

Will take a look once you provide more detail, and when I have free time.

**Comment by flameddd** [comment]
NO, I dont think so. I think `puppeteer` DONT support it.

**Comment by NilsBaumgartner1994** [comment]
Yes I know, but I think it would be nice to just „mock“ it and add a default bar with sharp on top?
